test/NuGet.Core.Tests/Microsoft.Build.NuGetSdkResolver.Test/RestoreRunnerExTests.cs

```diff
@@ -22,15 +23,14 @@ public class RestoreRunnerExTests
         public async Task RestoreRunnerEx_WithExistingPackage_DoesNotCreateAnyAssetsAsync()
         {
             using (var pathContext = new SimpleTestPathContext())
-            using (var context = new SourceCacheContext())
             {
                 // Arrange
                 var packageId = "x";
                 var packageVersion = "1.0.0";
                 var logger = new TestLogger();
                 var library = new LibraryIdentity(packageId, NuGetVersion.Parse(packageVersion), LibraryType.Reference);
 
-                var package = new SimpleTestPackageContext(packageId, packageVersion);
+                var package = new PackageIdentity(packageId, NuGetVersion.Parse(packageVersion));
```

Comment:
Is this also a rebase error?  It's not clear to me how this this relevant to onboarding to package source mapping

Reply:
I reverted it. It's just nice to have thing as part of fixing tests, things should still work without this change.